src/fillTank.test.js

```diff
@@ -1,11 +1,58 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
+  const customer = {
+    money: 3000,
+    vehicle: {
+      maxTankCapacity: 40,
+      fuelRemains: 8,
+    },
+  };
+  const fuelPrice = 4.8;
 
-  it('should ', () => {
+  it("should be declared", () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it("should round the poured amount to the tenth part", () => {
+    fillTank(customer, fuelPrice, 20.78);
+
+    expect(customer.vehicle.fuelRemains).toStrictEqual(28.7);
+  });
+
+  it("should order a full tank if the amount is not given", () => {
+    fillTank(customer, fuelPrice);
+
```

Comment:
The test 'should order a full tank if the amount is not given' also changes the initial state of the customer object. A separate customer object should be created for this test as well.